test/src/test/java/hudson/model/FileParameterValueTest.java

```diff
@@ -318,15 +326,23 @@ public void fileParameter_canStillUse_internalHierarchy() throws Exception {
 
         // and reachable using request
         JenkinsRule.WebClient wc = j.createWebClient();
+
+        LOGGER.info("getting ws");
         HtmlPage workspacePage = wc.goTo(p.getUrl() + "ws");
```

Comment:
although one way to solve may `await().until(..)...` so it retries a couple of times, and hopefully that solves. but the noticed delays (log in pr description) indicate something is wrong.. 🤔

Reply:
explicitly disabling the javascript makes the test faster...